requirements/base.in

```diff
@@ -24,6 +24,7 @@ django-better-admin-arrayfield
 # API libraries
 drf-extra-fields
 drf-writable-nested
-notifications-api-common
 humanize
 drc-cmis
+
+commonground-api-common[testutils]
```

Comment:
I think we should call the optional dependency something else for this project since it's not used for testing purposes (but in the actual application). We should do this through a separate issue though.